@HastD 

```

[17:05:18 g.i/s/sericea-main-hardened:pr-1526-43] => Compiling targeted trivalent module
[17:05:18 g.i/s/sericea-main-hardened:pr-1526-43] => trivalent.te:318:ERROR 'unrecognized character' at token ''' on line 20046:
[17:05:18 g.i/s/sericea-main-hardened:pr-1526-43] => '
[17:05:18 g.i/s/sericea-main-hardened:pr-1526-43] => #line 318
[17:05:18 g.i/s/sericea-main-hardened:pr-1526-43] => /usr/bin/checkmodule:  error(s) encountered while parsing configuration
[17:05:18 g.i/s/sericea-main-hardened:pr-1526-43] => make: *** [/usr/share/selinux/devel/include/Makefile:157: tmp/trivalent.mod] Error 1
```

--- a/files/scripts/selinux/trivalent/trivalent.te
+++ b/files/scripts/selinux/trivalent/trivalent.te
@@ -315,6 +315,13 @@ allow trivalent_script_t dosfs_t:filesystem remount;
 allow trivalent_script_t null_device_t:chr_file mounton;
 allow trivalent_script_t trivalent_t:process2 { nosuid_transition nnp_transition };
 
+# Do not audit attempts to read /proc/PID for other processes.
+optional_policy(`
+	gen_require(`
+		attribute domain;
+	')
+	dontaudit trivalent_script_t domain:dir getattr;
+')
 
 # xwayland/nvidia
 xserver_exec(trivalent_script_t)

